front/app/components/Form/Components/FormWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+
+interface Props {
+  formId: string | undefined;
```

Comment:
By making it `formId: string | undefined` rather than `formId?: string`, you make sure that the `formId` is always explicitly passes. So doing `<FormWrapper />` is not allowed, you have to do `<FormWrapper formId={undefined} />`.

I extracted this component because I thought it was necessary, but in retrospect it was not. In any case it's not that important so I will just leave it

Reply:
Yes, I understand that difference. But an explicit id with an undefined value seems unusual (I have no idea why you would do that and not make sure you pass a string) if you want an id. So, the allowed value should be a string, IMO. Anyway, I still think if formId is not used for anything, we should just remove it.

Reply:
Well, it was in the original code, and there is was also typed like this. I didn't have time to look into everything unfortunately, and I don't want to just change things without knowing that it's not going to break something